packages/config/src/android/__tests__/NotificationsManifest-test.ts

```diff
@@ -0,0 +1,75 @@
+import { resolve } from 'path';
+
+import { ExpoConfig } from '../../Config.types';
+import { getMainApplication, readAndroidManifestAsync } from '../Manifest';
+import {
+  applyAndroidManifestChanges,
+  META_DATA_NOTIFICATION_ICON,
+  META_DATA_NOTIFICATION_ICON_COLOR,
+  NOTIFICATION_ICON_COLOR_RESOURCE,
+  NOTIFICATION_ICON_RESOURCE,
+} from '../Notifications';
+
+const fixturesPath = resolve(__dirname, 'fixtures');
+const sampleManifestPath = resolve(fixturesPath, 'react-native-AndroidManifest.xml');
+const notificationConfig: ExpoConfig = {
+  name: 'lol',
+  slug: 'lol',
+  notification: {
+    icon: '/app/assets/notificationIcon.png',
+    color: '#00ff00',
+  },
+};
+
+describe('Applies proper Android Notification configuration to AndroidManifest.xml', () => {
```

Comment:
put this in a separate file because mocking `fs` was causing some issues for `Manifest. readAndroidManifestAsync`